thinkaloud.md

```diff
@@ -0,0 +1,84 @@
+## Vision
+On a high level, we firmly view fairness as a socio-technical challenge, so we want to provide techniques that enable humans to navigate fairness trade-offs in a manner that makes most sense for their context. Because of this, our package includes not only algorithm and metrics, but also the fairness assessment dashboard--and we hope that the development of UX for fairness assessment / unfairness mitigation will be our key area of contribution [fairlearn#311](https://github.com/fairlearn/fairlearn/issues/311#issuecomment-592716247).  Historically, the project started as a Python package that implements algorithms for mitigating unfairness in supervised machine learning.
+
+Fairness is fundamentally a sociotechnical challenge. Many aspects of fairness, such as justice and due process, are not captured by quantitative fairness metrics. Furthermore, there are many quantitative fairness metrics which cannot all be satisfied simultaneously. Our goal is to enable humans to assess different mitigation strategies and then make trade-offs appropriate to their scenario ([README.md](https://github.com/fairlearn/fairlearn#what-we-mean-by-fairness), [fairlearn#244](https://github.com/fairlearn/fairlearn/issues/244#issuecomment-575309920)).  To this point, 
+
+One big source of concern for us is how the issue of fairness is presented in our package (documentation and any other materials) - we do not want to enable so-called 'fairwashing' of ML algorithms ([fairlearn#406](https://github.com/fairlearn/fairlearn/issues/406#issuecomment-623541578)).
+
+While we have included example notebooks, these often emphasis the mechanics of interacting with the API rather than actual use cases dealing with fairness as a sociotechnical challenge ([fairlearn-proposals#2](https://github.com/fairlearn/fairlearn-proposals/issues/2#issuecomment-594941077)).  Moving forward, any documentation and examples of a fairness toolkit should take into account an entire range of concerns due to the sociotechnical nature of fairness. The showcased notebooks will provide the space to cover scenarios in depth. The focus is not only on showing example usage of the Fairlearn toolkit, but on how to approach fairness in ML in general. We may want to add a scenario even if it contains only few of Fairlearn's capabilities, but it otherwise demonstrates a great example of how to build AI responsibly ([fairlearn-proposals#8](https://github.com/fairlearn/fairlearn-proposals/pull/8/files#diff-21173c05e65dcf529aa6c8cbc26bf0e9R104)).
```

Comment:
@romanlutz process comment: thanks!  I've read this comment but this is on my fork and I really want to make sure you all think this is productive to do together before we discuss.

I'd love to avoid any discussion happening in my repo, so will close this PR (I should have earlier to make this intent clearer).  If y'all on the team think this is a productive step to take, please comment in https://github.com/fairlearn/fairlearn-proposals/issues/2#issuecomment-624736295, and I will open a PR on the real repo :)

I will also copy over any old stuff you happened to already comment on here as well, sorry for the mixup.